service/prometheus/group.go

```diff
@@ -0,0 +1,51 @@
+package prometheus
+
+import (
+	"k8s.io/client-go/pkg/api/v1"
+)
+
+// FilterServices takes a list of Kubernetes Services,
+// and returns a list of valid Services.
+func FilterServices(services []v1.Service) []v1.Service {
```

Comment:
Based on the godoc the function should be called `FilterInvalidServices` because it is removing the invalid, or unannotated, services out.

Reply:
Done